rebar.config.erl

```diff
@@ -28,7 +28,7 @@ plugins() ->
 
 test_deps() ->
     [ {bbmustache, "1.10.0"}
-    , {emqx_ct_helpers, {git, "https://github.com/emqx/emqx-ct-helpers", {tag, "1.3.0"}}}
+    , {emqx_ct_helpers, {git, "https://github.com/emqx/emqx-ct-helpers", {tag, "1.3.3"}}}
```

Comment:
`emqx_ct_helper` 1.3.3 and 1.3.2 are the same commit id, and I recommend removing 1.3.3 tags
I've updated the version of `emqx_ct_helper` in `rebar.config` to 1.3.2, and the code has been merged into dev/v4.3.0

Reply:
it looks like 1.3.2 was force updated ? anyway tag 1.3.3 has been deleted.